src/index.js

```diff
@@ -0,0 +1,114 @@
+import 'dotenv/config'
+
+import Discord, { MessageActionRow, Modal, TextInputComponent, Formatters } from 'discord.js'
+import { token } from './config'
```

Comment:
Besides eslint formatting and moving the file, these are the only lines I actually changed in this file and I didn't really change them. I just made them `import`s instead of `require`s and combined your `discord.js` into a single import.

Combining them into a single import should be automatically done for you now with the new ESLint config... I think. It might be a TS thing. I've been on TS for about 4yrs now, so I can't really remember what all are TS things and what are JS things. 😂